src/components/homepage/GetStartedTerminal.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent } from "react";
+import 'react-animated-term/dist/react-animated-term.css';
+import "./GetStartedTerminal.overrides.css";
+import Terminal from 'react-animated-term';
+
+
+const spinner = ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'];
+const cmdStep = (title:string) => ({
+  text: title,
+  cmd: true,
+  delay: 10
+})
+
+const spinningStep = (title:string, repeatCount = 0, frameDelay = 10) => ({
+  text: `✔ ${title}`,
+  cmd: false,
+  repeat: repeatCount > 0,
+  repeatCount: repeatCount,
+  frames: spinner.map((spinnerItem) => ({
+    text: `${spinnerItem} ${title}...`,
+    delay: frameDelay
+  }))
+})
+
+const termLines = [
+  cmdStep("capact env create kind"),
+  spinningStep("Creating cluster `kind-dev-capact`"),
+  cmdStep("capact install"),
+  spinningStep("Installing Capact on cluster"),
+  cmdStep("capact login https://gateway.capact.local -u user -p pass"),
+  {
+    text: '✔ Login succeeded',
+    delay: 10
+  },
+  cmdStep("# You're all set! Enjoy using Capact!"),
+]
+
+export const GetStartedTerminal: FunctionComponent = () => {
+  console.log(termLines);
```

Comment:
do we need that console log? I was for debugging, or it's some hack :D?

Reply:
It was for debugging, sorry 😄